doc/classes/Node3D.xml

```diff
@@ -145,7 +145,7 @@
 			<param index="0" name="axis" type="Vector3" />
 			<param index="1" name="angle" type="float" />
 			<description>
-				Rotates the local transformation around axis, a unit [Vector3], by specified angle in radians.
+				Rotates the local transformation around the parent's local axis, a unit [Vector3], by specified angle in radians.
```

Comment:
This isn't correct, or at least it's ambiguous, this implies that the axis is an axis of the parent

This should be done in a separate PR as it's unrelated to this

Reply:
`rotate` rotates the child object around the parent's local axis. If there's no parent, `rotate` works exactly like `global_rotate`.

`rotate_object_local` rotates around the object's own axis.

https://github.com/godotengine/godot/assets/37383316/ae6a095b-c314-4adc-82ed-174e356b9d05

I'm going to tweak a little this description to make it more obvious.

minimal example attached.

[rotation-example.zip](https://github.com/godotengine/godot/files/14898988/rotation-example.zip)

Reply:
Regardless, this belongs in its own PR, it's unrelated to this, further it's misleading, it is equivalent to `get_transform().rotate(...)`